src/integration-test/java/io/aiven/connect/jdbc/SchemaRegistryContainer.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.connect.jdbc;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.utility.Base58;
+
+public final class SchemaRegistryContainer extends GenericContainer<SchemaRegistryContainer> {
```

Comment:
This class looks familiar. Maybe move it to commons?

Reply:
True, we need to put it in common ... There are a lot of open questions how to it better ...